libr/anal/d/cc-avr-8

```diff
@@ -0,0 +1,26 @@
+default.cc=avr
+
+avr=cc
+cc.avr.name=avr
+cc.avr.argn=r26
```

Comment:
to make things simple we can just assume that all arguments will be of size 1 byte
then 
arg1=r26
arg2=r25
...
...
...
arg7=r18
for argn If arguments are put in memory (right to left) like in cdecl it will be `stack` but if it is put left to right (the same order you will pass it to function in C/C++ ) then it will be `stack_rev`